spec/fixtures/gitops-deploy-files/ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: helloworld-rubyapp-ingress
+  namespace: make-gitops-namespace-test-dev
+spec:
+  tls:
+  - hosts:
+    - make-gitops-namespace-test-dev.apps.gitops-test.cloud-platform.service.justice.gov.uk
```

Comment:
This is needs to be a variable and will not work on clusters not named `gitops-test`

Reply:
This file is used for comparison not execution.